R/format_model.R

```diff
@@ -10,7 +10,11 @@
 format_model <- function(model) {
   info <- insight::model_info(model)
 
-  type <- ""
+  if(all(insight::find_parameters(model, flatten = FALSE) == "(Intercept)")){
+    type <- "constant (intercept-only) "
+  } else{
```

Comment:
I think for brms, the intercept has a different name / spelling (without parantheses).

Reply:
Thanks, I'll try to keep that in mind!

I'll first try to fix it for base, lme4 and rstanarm, and then once we're back to where we were we can extend ☺️